Marlin/pins_ULTIMAKER.h

```diff
@@ -79,21 +80,33 @@
 #define E1_DIR_PIN         47
 #define E1_ENABLE_PIN      48
 
+#define E2_STEP_PIN        49
+#define E2_DIR_PIN         47
+#define E2_ENABLE_PIN      48
+
+#define E3_STEP_PIN        E2_STEP_PIN
+#define E3_DIR_PIN         E2_DIR_PIN
+#define E3_ENABLE_PIN      E2_ENABLE_PIN
+
```

Comment:
I'm not sure of the overall effect of having E1, E2, and E3 all point to the same pins, all the time. Please hold off on this change. If this is important to making `SINGLENOZZLE_SWITCHING_EXTRUDER` work, then we need to find a more universal method that works for all pins files. (I'm sure we shouldn't go into all the pins files and add duplicated E pins, as here.)